scripts/run_mypy_checks.py

```diff
@@ -626,6 +626,8 @@
     'jobs/transforms/question_validation_test.py',
     'jobs/transforms/topic_validation.py',
     'jobs/transforms/topic_validation_test.py',
+    'jobs/transforms/subtopic_validation.py',
+    'jobs/transforms/subtopic_validation_test.py',
```

Comment:
@hardikkat24 Are we expecting to add new files here?

Reply:
@DubeySandeep Adding new files to the denylist must be avoided. 
@YBCS Can you please add type annotations? Thanks!

Reply:
Done PTAL @DubeySandeep @hardikkat24